src/etc/inc/plugins.inc.d/core.inc

```diff
@@ -426,7 +426,7 @@ function core_xmlrpc_sync()
     $result[] = array(
         'description' => gettext('Static Routes'),
         'help' => gettext('Synchronize the Static Route configuration and Gateways to the other HA host.'),
-        'section' => 'staticroutes,gateways',
+        'section' => 'staticroutes,gateways,OPNsense.Gateways',
```

Comment:
@swhite2 @AdSchellevis this is marginally related to the PR, but I have to ask :)

Here we keep "gateways" because the code can deal with it.... but what do we actually do in the backup import now with these sections all being marked as "importable"? Because I've been thinking of the alias situation... Should we clear the sections not given in the backup? It would allow a direct old alias import since it forces migration... I suppose the same is true for "gateways" key from an older install when "OPNsense.Gateways" was already created by the default migration on a newer version..

Reply:
pew pew? https://github.com/opnsense/core/commit/b8bbb00da7

works for me on legacy "aliases" backup ;)